### PR TITLE
feat(settings): moved settings profile and changed to be trading limits and as Flyout

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
This PR implements flyout for trading limits page
image

## Testing Steps (optional)
Login and click on gear icon on top right corner, then click on Trading Limits from dropdown, new flyout should be visible

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+`
+const Errors = styled(ErrorCartridge)`
+  font-size: 12px;
+`
+const Blue = styled(BlueCartridge)`
+  font-size: 12px;
+`
+const Orange = styled(OrangeCartridge)`
+  font-size: 12px;
+`
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx:81]: please modify the base cartridge to take a font-size prop and pass the size in that way

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/model.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    [ITEMS.RECEIVE]: false,
+    [ITEMS.SWAP]: false,
+    [ITEMS.CASH_ACCOUNT]: false,
+    [ITEMS.BUY_WITH_A_CARD]: false,
+    [ITEMS.DEPOSIT_AND_WITHDRAWAL]: false,
+    [ITEMS.EARN_INTEREST]: false
+  },
+  [TIER_TYPES.SILVER]: {
+    [ITEMS.SEND]: true,
+    receive: true,
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/model.ts:30]: `[ITEMS.RECEIVE]: true`?

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/model.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    [ITEMS.SWAP]: true,
+    [ITEMS.CASH_ACCOUNT]: false,
+    [ITEMS.BUY_WITH_A_CARD]: false,
+    [ITEMS.DEPOSIT_AND_WITHDRAWAL]: false,
+    [ITEMS.EARN_INTEREST]: false
+  },
+  [TIER_TYPES.GOLD]: {
+    [ITEMS.SEND]: true,
+    receive: true,
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/model.ts:39]: `[ITEMS.RECEIVE]: true?`

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/model.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    [ITEMS.SWAP]: true,
+    [ITEMS.CASH_ACCOUNT]: true,
+    [ITEMS.BUY_WITH_A_CARD]: true,
+    [ITEMS.DEPOSIT_AND_WITHDRAWAL]: true,
+    [ITEMS.EARN_INTEREST]: true
+  },
+  [TIER_TYPES.SILVER_PLUS]: {
+    [ITEMS.SEND]: true,
+    receive: true,
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/model.ts:48]: `[ITEMS.RECEIVE]: true?`

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+      </ContentItem>
+      <ContentItem>
+        <Icon name='percentage' color='blue600' size='20px' />
+
+        <Text
+          color='grey900'
+          size='16px'
+          weight={600}
+          style={{ marginLeft: '20px' }}
+        >
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx:390]: this exact same Text styling is repeated 7 times, please create a single styled-component

File: packages/blockchain-wallet-v4-frontend/src/modals/index.tsx [hunk 6]
```diff
@@ -101,6 +101,8 @@ const TwoStepGoogleAuthenticator = React.lazy(() =>
 const TwoStepSetup = React.lazy(() => import('./Settings/TwoStepSetup'))
 const TwoStepYubico = React.lazy(() => import('./Settings/TwoStepYubico'))
 
+const TradingLimits = React.lazy(() => import('./Settings/TradingLimits'))
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/index.tsx:104]: delete empty line above so its grouped properly

**Review by schnogz — CHANGES_REQUESTED** [review]

**Comment by schnogz** [comment]
This does not work for new wallets.  The first time the modal opens and then closes right away.  It then appears as though the user has started the silver flow (although they havent) and will not reopen.

File: packages/blockchain-info-components/src/Images/img/tier-silver.svg [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="2" y="0" width="28" height="32">
```
Comment (yichok) [inline packages/blockchain-info-components/src/Images/img/tier-silver.svg:2]:
```suggestion
<mask id="mask0" mask-type="alpha" maskUnits="userSpaceOnUse" x="2" y="0" width="28" height="32">
```

**Review by yichok — CHANGES_REQUESTED** [review]

**Review by yichok — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx (hunk 1 above)
Comment (yichok) [reply packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx:74]: Hi

File: packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/index.tsx [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+import { UserDataType } from 'data/modules/types'
+import React, { PureComponent } from 'react'
+
+import { actions } from 'data'
+import { ModalPropsType } from './../../types'
+
+import { getData } from './selectors'
+
+import Loading from './template.loading'
+import Success from './template.success'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/index.tsx:18]: can we group and sort these imports by the new standards?
```
group 1 -> 3rd party imports
group 2 -> aliased imports
group 3 -> relative import
```

**Review by schnogz — CHANGES_REQUESTED** [review]

**Pull request merged**